examples/sdxl.py

```diff
@@ -0,0 +1,1107 @@
+# This file incorporates code from the following:
+# Github Name                    | License | Link
+# tinygrad/tinygrad              | MIT     | https://github.com/tinygrad/tinygrad/blob/64cda3c481613f4ca98eeb40ad2bce7a9d0749a3/LICENSE
+# Stability-AI/generative-models | MIT     | https://github.com/Stability-AI/generative-models/blob/fbdc58cab9f4ee2be7a5e1f2e2787ecd9311942f/LICENSE-CODE
+# mlfoundations/open_clip        | MIT     | https://github.com/mlfoundations/open_clip/blob/58e4e39aaabc6040839b0d2a7e8bf20979e4558a/LICENSE
+
+from tinygrad.tensor import Tensor, dtypes
```

Comment:
from tinygrad import Tensor, TinyJit, dtypes

also from tinygrad.helpers import trange should work

Reply:
4aef32d